### PR TITLE
Refactor platform api operator

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -161,33 +165,33 @@ install: ## Install all resources (CR/CRD's, RBCA and Operator)
 	@echo ....... Set environment variables ......
 	- export DEPLOY_DIR=deploy/crds
 	- export WATCH_NAMESPACE=${NAMESPACE}
-	@echo ....... Creating namespace .......
-	- kubectl create namespace ${NAMESPACE}
+	# @echo ....... Creating namespace .......
+	# - kubectl create namespace ${NAMESPACE}
```
Comment (bitscuit) [inline Makefile]: why not just remove the commented lines?

Comment (cheewaio) [reply Makefile]: sure

**Pull request merged**